WordPress/Classes/ViewRelated/Plugins/PluginDetailViewHeaderCell.swift

```diff
@@ -0,0 +1,67 @@
+import UIKit
+import WordPressShared
+import Gridicons
+
+class PluginDetailViewHeaderCell: UITableViewCell {
+
+    open func configureCell(_ directoryEntry: PluginDirectoryEntry) {
+
+        if let banner = directoryEntry.banner {
+            headerImageView?.downloadImage(banner, placeholderImage: nil)
+        } else {
+            headerImageView?.isHidden = true
+        }
+
+        let iconPlaceholder = Gridicon.iconOfType(.plugins, withSize: CGSize(width: 40, height: 40))
+        iconImageView?.downloadImage(directoryEntry.icon, placeholderImage: iconPlaceholder)
+
+        nameLabel?.text = directoryEntry.name
+
+        let author = directoryEntry.author!
```

Comment:
Why is that `!` there? If there's no chance an entry has a missing author, the model should not be optional, if it can be nil, it should not crash

Reply:
True, it was leftover when I was experimenting with stuff that I missed when cleaning up. Fixed!